src/airbrake.cr

```diff
@@ -49,22 +49,29 @@ module Airbrake
   class Config
     property :project_id
     property :project_key
+    property :host
+    property :port
+    property :secure
+
+    def uri
+      self.host   ||= "airbrake.io"
+      self.port   ||= 443
+      scheme = (self.secure.nil? || self.secure) ? "https" : "http"
+
+      URI.new(scheme, host, port, "/api/v3/projects/#{project_id}/notices", "key=#{project_key}").to_s
```

Comment:
We could deduce the `secure` option from this (and remove the option). For example:

```ruby
def uri
  # ...
  URI.new(scheme, host, port, "/api/v3/projects/#{project_id}/notices", "key=#{project_key}")
end
```

Then, when POSTing, we could use this to determine the protocol: `Airbrake.config.uri.is_a?(URI::HTTPS)`. This would work in Ruby, I'm not sure about Crystal.